update-tests.sh

```diff
@@ -0,0 +1,34 @@
+git submodule init
+git submodule update
+cd wit-bindgen
```

Comment:
I was originally hoping that this project wouldn't depend on `wit-bindgen` which would allow `wit-bindgen` itself to depend on this project. If you'd rather stick to this route though then I'll update `wit-bindgen` to use the Wasmtime support for components and remove the js-host dependency. 

In the end I just want to remove the js-host dependency from `wit-bindgen` to not duplicate it there from here. I was hoping to use JS to run host tests since it lends itself to a bit speedier iteration times since Wasmtime doesn't have to be built, nor the embedding for Wasmtime. 

I realize though that writing tests for the component model is not easy right now. You can see [in my PR to update `wasmtime-py`](https://github.com/bytecodealliance/wasmtime-py/pull/121) how I've got a bunch of handwritten `*.wat` files but they're a pain to update and not easy to extend.

So, essentially, would you  prefer to stick to this testing strategy? If so I'll drop JS from wit-bindgen and run tests with Wasmtime. If not then the tests here would be rewritten to not use `wit-bindgen` and I'll drop JS from wit-bindgen to use this project.

Reply:
A major drawback of precompiled tests, which is why I do not use them, is that they basically cannot be debugged, modified, or recreated. Debugging is often crucial when the generator is changing, for example. Modifying becomes important when bugs are fixed or new features are added and tests want to be added to reflect this. Creating new tests is similar for new features.

If the JS integration here relies on wit-bindgen for tests that means that if wit-bindgen were to rely on JS for running tests that's a circular dependency. For breaking changes, like the ones that just happened, there would be no way to actually update the repositories and keep CI working. That's why I want to maintain a non-circular sequence of steps to landing breaking changes. It sounds though like I should drop the JS host generator and bring back the Wasmtime support for running the tests.

Reply:
That's a great point, and I can get behind moving towards a future where all the tests are compiled internally. Perhaps a future goal to aim for might be to have all the tests written in componentize-js themselves when that is ready.

Reply:
I think it's reasonable to see how this plays out over time. I think a good number of tests are eventually going to be along the lines of "test this unsual shape of a component" which wouldn't be something a tool like `componentize-js` could do, but there's a lot of other runtime-y tests which could be tested too.

No matter what happens I find this aspect of the component model particularly difficult because it's not trivial to create components yet and it's also not entirely trivial to run components, so testing everything in tandem often feels a bit janky.

Reply:
Certainly let's continue to reassess here.